aspnetcore/mvc/views/overview.md

```diff
@@ -176,12 +185,11 @@ namespace WebApplication1.ViewModels
 > [!NOTE]
 > Nothing prevents you from using the same classes for both your viewmodel types and your business model types. However, using separate models allows your views to vary independently from the business logic and data access parts of your app. Separation of models and viewmodels also offers security benefits when models use [model binding](xref:mvc/models/model-binding) and [validation](xref:mvc/models/validation) for data sent to the app by the user.
 
-
 <a name="VD_VB"></a>
 
-### Weakly-typed data (ViewData and ViewBag)
+### Weakly-typed data (ViewData, ViewData attribute, and ViewBag)
 
-Note: `ViewBag` isn't available in the Razor Pages.
+Note: `ViewBag` isn't available in Razor Pages.
```

Comment:
I suggest making this a bit more prominent by making it a NOTE widget.

Reply:
Ok, I'll change it, but @Rick-Anderson (IIRC here) said that there is some indication that readers read right past our NOTE blocks. If there's more to that story, I wish we had some data to look at ... Are readers *really* just skipping our NOTE notes but actually reading our plain-text notes? Whatever the case with that, I'll add the NOTE blocks here.

Reply:
I prefer not to have notes. Just give them the info without all that distracting chrome.

Reply:
That's a lot of Note chrome on this page. Why convert all the others?

Reply:
I'm OK either way, but I prefer to minimize Note chrome. The security one for sure could use a note.

Reply:
> I prefer not to have notes. Just give them the info without all that distracting chrome.

That's what I do when I write new topics. I try to just provide the info in-text wherever possible and avoid them generally.

Let's just do the security one. My problem with notes isn't even so much about readers skipping them (although that's a good reason not to use them). I agree that they are generally distracting and unnecessary. Even using the word "Note: " before the note is kind'a silly ...

Note: You need to know this important tidbit.

or

**Note:** You need to know this important tidbit.

I mean every sentence in the topic could have that! lol Emphasis is probably better like this ...

*You need to know this important tidbit.*

... and move on.

> Why convert all the others?

That's what happens when the contractor gets conflicting direction! :smile: lol

This isn't the first time a review has come back 'make a formal NOTE out of that note.'